xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp

```diff
@@ -72,6 +73,26 @@ bool CGUIDialogPVRGuideInfo::OnClickButtonOK(CGUIMessage &message)
   return bReturn;
 }
 
+bool CGUIDialogPVRGuideInfo::OnClickButtonChannelGuide(CGUIMessage &message)
+{
+  bool bReturn = false;
+
+  if (message.GetSenderId() == CONTROL_BTN_CHANNEL_GUIDE)
+  {
+    if (!m_progItem || !m_progItem->HasPVRChannel())
+    {
+      /* invalid channel */
+      CGUIDialogOK::ShowAndGetInput(CVariant{19033}, CVariant{19067});
```

Comment:
Could you please add the text for the two resource identifiers as a comment, like we usually do?